elucidation-bundle/src/test/java/org/kiwiproject/elucidation/server/jobs/PollForEventsJobTest.java

```diff
@@ -77,9 +74,9 @@ void firstPoll() {
         var sinceCaptor = ArgumentCaptor.forClass(Long.class);
 
         when(SERVICE.listEventsSince(anyLong())).thenReturn(newArrayList(
-                newConnectionEvent(A_SERVICE_NAME, Direction.INBOUND, MSG_FROM_ANOTHER_SERVICE),
-                newConnectionEvent(A_SERVICE_NAME, Direction.OUTBOUND, MSG_TO_ANOTHER_SERVICE),
-                newConnectionEvent(A_SERVICE_NAME, Direction.OUTBOUND, IGNORED_MSG)
+                newConnectionEvent(TestConstants.A_SERVICE_NAME, Direction.INBOUND, TestConstants.MSG_FROM_ANOTHER_SERVICE),
```

Comment:
Did you want to statically import these, like they were previously? Ditto for any other usages in this or any other test.